core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -1043,6 +1043,18 @@ public PluginWrapper getInstalled() {
             return pm.getPlugin(name);
         }
 
+        @Exported
```

Comment:
Unnecessary?

Reply:
I assume the fact that there is a jelly file that invokes it doesn't imply that the method has to be annotated with "Exported", right? I wasn't sure how this was being used.

If that is not a requirement, then there is no need to have it, so I'll be happy to remove it.

Reply:
This is for the "REST" API, i.e. `/pluginManager/api` or `/updateCenter/api`, don't remember which, and entirely optional.

Reply:
Thanks for the explanation, Daniel 👍

Reply:
Since there have been no comments from others on this, I've applied @daniel-beck 's suggestion and removed the `@Exported` annotation. Definitely not necessary for now.